cosmos/stratos.json

```diff
@@ -0,0 +1,56 @@
+{
+  "rpc": "https://rpc.thestratos.org",
+  "rest": "https://rest.thestratos.org",
+  "chainId": "stratos-1",
+  "chainName": "Stratos Network",
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/stratos/chain.png",
+  "walletUrl": "https://wallet.keplr.app/chains/stratos",
+  "walletUrlForStaking": "https://wallet.keplr.app/chains/stratos",
+  "nodeProvider": {
+    "name": "Stratos",
+    "email": "info@thestratos.org",
+    "website": "https://thestratos.org"
+  },
+  "bip44": {
+    "coinType": 606
```

Comment:
@BoThe1K Hello!
Normally, the coinType for chains used with evm is 118, so why did you choose 606?

Reply:
what do you mean by 'the coinType for chains used with evm is 118'? from slip44 I see 118 used by ATOM, how is that related to evm? 
our coinType is registered in slip44 as 606, refer to https://github.com/satoshilabs/slips/blob/master/slip-0044.md?plain=1#L637 
So I am a bit confused by your question. Are this bip44 coinType used differently by keplr?

Reply:
@alexstratos Oh may bad.
It was 60, not 118.

Reply:
please correct me if my understanding is wrong.
the field of 
 "bip44": {
    "coinType": 606
 }
 is used to generate the wallet key to interact with our chain's cosmos-sdk functionality instead of evm functionality. if that's the case, this field should be 606. 
 if my understanding is wrong(meaning this field is used for evm functionality), this should be 60.

Reply:
@HeesungB if there is some concerns regarding `"features": ["eth-address-gen", "eth-key-sign"]`, we could remove these lines. Our chain working both with tendermint/cosmos api and separatelly with our web3. The metamask keys will be derived from 60 path (could not be changed) and still could be used though the web3 api (as it do not make any impact on signer recovery).

Reply:
@BoThe1K @alexstratos If Keplr also supports EVM, we recommend a coin type of 60 for the derived path.
I am concerned that using 606 will cause problems even when using Ledger.

Reply:
ah, I understand your concerns now, that cosmos ledger app only allowed 118 or 60 right now. 
Ok, we changed to 60 for coinType to make it adapt to cosmos ledger app,